js/utils.js

```diff
@@ -17,6 +17,55 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
   };
   var computedFieldClashes = [];
 
+  function setLineClamps () {
+    setClampHeight()
+      .then(function (elements) {
+        Superclamp.register(elements);
+        return Promise.resolve(elements);
+      })
+      .then(function (elements) {
+        removeClampHeight(elements);
+      })
+      .catch(function (selector) {
+        console.error('Fail to clamp line in the selector ' + selector);
+      });
+  }
+
+  function setClampHeight (selector) {
+    selector = selector || '[data-line-clamp]';
+
+    return new Promise(function (resolve, reject) {
+      var elements = document.querySelectorAll(selector);
+      
+      if (!elements.length) {
+        reject(selector);
+      }
+
+      for (var i = 0; i < elements.length; i++) {
+        var element = elements[i];
+        var lineHeightValue = window.getComputedStyle(element)
```

Comment:
What does the `.match(/[a-zA-Z]+|[0-9]+(?:\.[0-9]+|)/g)` part do?

Does it always return a number in pixels? If not, then what are the possible values and how are we dealing with each of the possible value types/units?

Reply:
@tonytlwu 

> Does it always return a number in pixels?

I've tested it with different values as `em` or `rem` and it always returns the value in pixels.

> What does the .match(/[a-zA-Z]+|[0-9]+(?:\.[0-9]+|)/g) part do?

This part splits the code to array with value and unit.